protocol/messenger.go

```diff
@@ -3167,7 +3167,7 @@ func (m *Messenger) syncChatRemoving(ctx context.Context, id string) error {
 	clock, chat := m.getLastClockWithRelatedChat()
 
 	syncMessage := &protobuf.SyncChatRemoved{
-		Clock: clock,
+		Clock: chat.DeletedAtClockValue,
```

Comment:
This one causes a test to fail 

```golang
func (s *MessengerInstallationSuite) TestSyncInstallation() {
```

So not sure if it's really correct to use this clock here